pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/extensible/channel/ServiceUnitStateDataTest.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.broker.loadbalance.extensible.channel;
+
+import static org.apache.pulsar.broker.loadbalance.extensible.channel.ServiceUnitState.Assigned;
+import static org.apache.pulsar.broker.loadbalance.extensible.channel.ServiceUnitState.Owned;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNull;
+import static org.testng.Assert.assertTrue;
+import org.testng.annotations.Test;
+
+@Test(groups = "broker")
+public class ServiceUnitStateDataTest {
+
+    @Test
+    public void testConstructors() {
+        ServiceUnitStateData data1 = new ServiceUnitStateData(Owned, "A");
+        assertEquals(data1.getState(), Owned);
+        assertEquals(data1.getBroker(), "A");
+        assertNull(data1.getSourceBroker());
+        assertTrue(data1.getTimestamp() > 0);
```

Comment:
You can use `assertj`'s assertion for better error message:

```java
import static org.assertj.core.api.Assertions.assertThat;

assertThat(data1.getTimestamp()).isGreaterThan(0);
```

Reply:
ditto other assertions.

Reply:
Updated.